app/admin/allocation_chart.rb

```diff
@@ -39,7 +39,8 @@ def build_allocation_status_cell(last_allocation)
       last_allocation: last_allocation,
       allocation_status: allocation_status
     ),
-    allocation_status: allocation_status
+    allocation_status: allocation_status,
+    last_allocation: last_allocation
```

Comment:
Why?

Reply:
Because I use this value to generate the path in `admin_allocation_path`